code/modules/wiremod/components/bci/hud/object_overlay.dm

```diff
@@ -88,6 +90,12 @@
 		QDEL_NULL(active_overlays[target_atom])
 		active_overlays.Remove(target_atom)
 
+	// Clear all overlays
+	if(COMPONENT_TRIGGERED_BY(signal_all_off, port))
+		for(var/atom/active_overlay in active_overlays)
+			QDEL_NULL(active_overlays[active_overlay])
+			active_overlays.Remove(active_overlay)
```

Comment:
```suggestion
		for(var/active_overlay in active_overlays)
			QDEL_NULL(active_overlays[active_overlay])
		active_overlays.Cut()
```
1. Is it possible for non-atoms to be stored in the overlay list?
2. If not, then we can remove the type check and use Cut instead for a very slight speed improvement (even though this is a cold path so it won't really matter that much, it just looks cleaner)

Reply:
I haven't looked at that, I'm guessing probably not.

This code was copied from the code that runs when the module is removed. I can dig into that though.